src/avram/database.cr

```diff
@@ -171,67 +177,23 @@ abstract class Avram::Database
     transactions.delete(Fiber.current.object_id)
   end
 
-  def table_names
-    tables_with_schema(excluding: "migrations")
-  end
-
-  def tables_with_schema(excluding : String)
-    select_rows <<-SQL
-    SELECT table_name
-    FROM information_schema.tables
-    WHERE table_schema='public'
-    AND table_type='BASE TABLE'
-    AND table_name != '#{excluding}';
```

Comment:
Do we need to account for this somewhere to avoid having the migrations table included in any of this?

Reply:
So, I'm not entirely sure why this condition was there to begin with since all the checks are pulling out specific database tables. So I guess there's two differences between master and this right now.
1. I am no longer excluding the migrations database
2. I am pull tall tables (which includes views) and not just `BASE TABLE`'s

Reply:
Cool with me. Maybe it was just to save on doing an extra query? But with this all being in one big query it probably doesn't matter.